server/src/main/java/org/apache/druid/client/CoordinatorServerView.java

```diff
@@ -162,7 +180,28 @@ private void serverAddedSegment(final DruidServerMetadata server, final DataSegm
         );
         segmentLoadInfos.put(segmentId, segmentLoadInfo);
       }
+
+      if (null != druidClientFactory) {
+        QueryRunner queryRunner = serverQueryRunners.get(server.getName());
+        if (null == queryRunner) {
+          DruidServer inventoryValue = baseView.getInventoryValue(server.getName());
+          if (inventoryValue == null) {
```

Comment:
nit: can you please either put `null` consistently on the left or right of these checks? My preference would be on the right since my gut says that is what the majority of the codebase is doing, but at least we should be consistent in the same function...

Reply:
I have tried to adhere with the general convention and moved null to the right.